### PR TITLE
Bring in @davidlukemt's changes to support VCF 5.1

### PR DESCRIPTION
**Pull request by ctreatma**
This PR replaces the existing contents of this repo with those of https://github.com/davidlukemt/tfm-eqix-metal-vcf.

This repo relies on older versions of things and may not work reliably (or at all?) in its current state.  The new contents support VCF and demonstrate usage of VRF Metal Gateways.

This will likely need some tweaks in follow-up PRs.

### CODE REVIEW

File: terraform.tfvars.example [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+## ESXi device common details
+esxi_management_subnet    = "255.255.255.0"        # Management Network Subnet Mask for VMK0
+esxi_management_gateway   = "172.16.11.1"          # Management Network Gateway for default TCP/IP Stack
+esxi_dns_server           = "172.16.1.1"           # DNS Server to be configured in ESXi
+esxi_domain               = "sfo.rainpole.io"      # Domain Name to be configured in ESXi FQDN along with name above
+esxi-mgmt_vlan            = "1611"                 # VLAN ID of Management VLAN for ESXi Management Network portgroup/VMK0
+esxi_ntp_server           = "172.16.1.1"           # NTP Server to be configured in ESXi
+esxi_password             = ""                     # Pre-hashed root password to be set for ESXi instances https://github.com/davidlukemt/tfm-eqix-metal-vcf?tab=readme-ov-file#custom-root-password
```
Comment (displague) [inline terraform.tfvars.example:125]:
```suggestion
esxi_password             = ""                     # Pre-hashed root password to be set for ESXi instances https://github.com/equinix-labs/terraform-eqix-metal-vcf?tab=readme-ov-file#custom-root-password
```

File: README.md [hunk 2]
```diff
@@ -1,153 +1,84 @@
-# VMware Cloud Foundation on Equinix Metal
-by: Benjamin Jenkins
+# tfm-eqix-metal-vcf
```
Comment (displague) [inline README.md:1]:
```suggestion
# terraform-equinix-metal-vcf
```

File: LICENSE [hunk 3]
```diff
@@ -1,339 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```
Comment (displague) [inline LICENSE:1]: replace with Apache?

Comment (ctreatma) [reply LICENSE]: Opted to keep the previous license instead, since I didn't need to remove this file in order to apply David's changes.

**Comment by ctreatma** [comment]
Merging the changes via a PR will make it harder to migrate in-progress work from the old repo to this one.  Per out-of-band discussion, this PR looks to be in good shape, so I'm going to close this and push these changes directly to `main` for a more workable history.

**Pull request closed without merging**